### ISSUE TITLE
Feature: Let users add new References

### ISSUE THREAD

**Issue by subwaystation** [lanagarmire/Granatum#1]
Hi there,
I would like to add a new genome reference to use with Granatum. This is not possible, yet?
Best,
Simon

**Comment by lanagarmire** [comment]
That is a good idea. Yes, we should consider it. Perhaps we need to
encourage user upload of genome interpretation of their preference.

What do you think, team?

**Comment by lanagarmire** [comment]
Yes I agree. So by genome reference you mean as opposed to RefSeq?

**Comment by subwaystation** [comment]
At the moment you provide Human and Mouse. And I would be able to add another organism?
Would that be hard to implement?

**Comment by lanagarmire** [comment]
One place that the reference genome is a must is gene set enrichment.
Other steps should be mostly fine. It may be tricky to do KEGG analysis in
other organisms, the user may need to be provide the physical mapping
between genes and pathway IDs.
